lib/utils.pm

```diff
@@ -114,6 +114,9 @@ sub unlock_if_encrypted {
     return unless get_var("ENCRYPT");
 
     if (check_var('ARCH', 's390x') && check_var('BACKEND', 'svirt')) {
+        # we need to reset the svirt console to extract the pty variable correctly
+        console('svirt')->reset;
+        select_console('svirt');
```

Comment:
IIUC the wrong location. It needs to be set whenever we reboot machines not here

Reply:
yeah - also discussed with @rwx788, makes sense to move to `redefine_svirt_domain`